src/EditorFeatures/TestUtilities/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -83,6 +84,9 @@ private CompletionOptions GetCompletionOptions()
             if (ShowNameSuggestions.HasValue)
                 options = options with { ShowNameSuggestions = ShowNameSuggestions.Value };
 
+            if (ShowNewSnippetExperienceFeatureFlag.HasValue)
+                options = options with { ShowNewSnippetExperience = ShowNewSnippetExperienceFeatureFlag.Value };
```

Comment:
You probably want the `ShowNewSnippetExperience` option here, instead of the feature flag. The actual user facing option is what ultimately controls the behavior.